packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -60,7 +62,7 @@ type BaseButtonCommonProps = {
   }>;
   type?: 'button' | 'reset' | 'submit';
   isLoading?: boolean;
-  accessibilityLabel?: string;
+  accessibilityProps?: Partial<AccessibilityProps>;
```

Comment:
didn't understand why this?

Reply:
Earlier all HTML props were getting spread no so we could do `{...makeAccessible({ /* accessibility props */ })}` and they were getting added (we did this at multiple places)

Now that we're not spreading all props. We need an alternative way to pass accessibility props to these base components. This is that alternative way.